### ISSUE TITLE
Does anyone encounter low accuraccy with profile faces

### ISSUE THREAD

**Issue by wangchust** [davidsandberg/facenet#460]
when detect a profile face, the recognition result is always wrong. Uh, Does anyone have any ideas??
My plan is to not detect profile face, but dont how to go on.

**Comment by RakshakTalwar** [comment]
Yes, you will get false positives with profile images. This is an issue with almost every face recognition system.
